src/mgr/Action.cc

```diff
@@ -120,3 +139,26 @@ Mgr::Action::fillEntry(StoreEntry* entry, bool writeHttpHeader)
         entry->complete();
 }
 
+void
+Mgr::OpenKidSection(StoreEntry *entry, const Format format)
+{
+    switch (format) {
+    case Format::yaml:
+        return storeAppendPrintf(entry, "---\nkid: %d\n", KidIdentifier);
+    case Format::informal:
+        return storeAppendPrintf(entry, "by kid%d {\n", KidIdentifier);
```

Comment:
> Pr description: Non-YAML non-aggregated reports in non-SMP instances are now framed using "by kid0 {...}" wrappers (for `squid -N`) or "by kid1 {...}" wrappers (for instances having one worker process and no rock diskers). This change makes YAML and non-YAML report framing more similar, but may affect existing report parsing automation.

I am still not sure we should change non-YAML framing as disclosed in the above paragraph, but I am not against that change.

Reply:
Francesco and I have updated Daft tests to cope with this PR changes -- CI functionality-tests check is now green.